### PR TITLE
Update what_do_i_need.md

### PR DESCRIPTION
**Pull request by Gamester17**
Update what_do_i_need.md

### CODE REVIEW

File: docs/getting_started/what_do_i_need.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ In order to run Zigbee2MQTT we need the following hardware:
 
 | Name | Price | Picture |
 | ------------- | ------------- | ------------- |
-| Supported Zigbee adapter | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
-| [Supported Zigbee device(s)](../information/supported_devices.md) | - | ![Zigbee devices](../images/xiaomi_sensors.jpg) |
-| PC or Raspberry Pi (*) to run bridge | - | ![Raspberry Pi](../images/pi.jpg) |
+| A supported [Zigbee adapter](../information/supported_adapters.md) (with Zigbee coordinator firmware) | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
```
Comment (Koenkk) [inline docs/getting_started/what_do_i_need.md:8]: "(with Zigbee coordinator firmware)" can be left out, it's not a requirement as below is explained how to do this.

File: docs/getting_started/what_do_i_need.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ In order to run Zigbee2MQTT we need the following hardware:
...
+| One or more [supported Zigbee device(s)](../information/supported_devices.md) | - | ![Zigbee devices](../images/xiaomi_sensors.jpg) |
+| A PC or a Raspberry Pi (to run the Zigbee2MQTT software) | - | ![Raspberry Pi](../images/pi.jpg) |
```
Comment (Koenkk) [inline docs/getting_started/what_do_i_need.md:10]: I think: "A computer (e.g. Raspberry Pi) to run the Zigbee2MQTT software on" is better

File: docs/getting_started/what_do_i_need.md (hunk 1 above)
Comment (Gamester17) [reply docs/getting_started/what_do_i_need.md]: OK

File: docs/getting_started/what_do_i_need.md (hunk 2 above)
Comment (Gamester17) [reply docs/getting_started/what_do_i_need.md]: OK

**Comment by Gamester17** [comment]
OK?

**Comment by Koenkk** [comment]
Yes, thanks!

**Pull request merged**